src/biokbase/narrative/jobs/jobmanager.py

```diff
@@ -195,10 +195,10 @@ def list_jobs(self):
 
             if not len(status_set):
                 return "No running jobs!"
-            status_set = sorted(status_set, key=lambda s: ObjectId(s['_id']).generation_time.timestamp())
+            status_set = sorted(status_set, key=lambda s: ObjectId(s['job_id']).generation_time.timestamp())
```

Comment:
This can just sort on s['created'] now, right?

Reply:
👍